libs/wingsdk/src/target-sim/app.ts

```diff
@@ -45,6 +45,9 @@ export class App extends Construct implements core.IApp {
     // write simulator.json file into workdir
     this.synthSimulatorFile(workdir);
 
+    // write tree.json file into workdir
+    core.synthesizeTree(this, workdir);
```

Comment:
Is this going to break the Wing Console?

Reply:
Probably. But I think it should be possible to update the console to work in a backwards-compatible way

Reply:
Backward compatibility is the responsibility of the "server". In our case it's the wing toolchain, not the Console.

The "right thing" to do here is to emit both the old format and the new format until the console is updated to support the new format.

Reply:
Ok, good point. 🙂 Updated to be backwards compatible... I tested out the changes with the latest version of the Wing console and it works as expected.